notebooks/amazon_forest_notebook.py

```diff
@@ -214,15 +210,32 @@
 
 # <markdowncell>
 
+# ## Choose Hyperparameters
+# 
+# Choose your hyperparameters below for training. 
+# 
+# If you'd like to create a learning rate annealing schedule you can set a series of learning rates with `lr_1, lr_2, lr_3` and corresponding number of epochs for each `epochs_1, epochs_2, epochs_3`. 
+# 
+# Alternatively, if you just want to run one training session at a fixed learning rate and num epochs you can just enter values for `lr_1` and `epochs_2` only.
```

Comment:
Following these instructions I'll be tempted to define only the following learning rates and epochs:
```
lr_1 = 0.001
epochs_1 = 10
```
Except that if I do that the code below will crash :P

It's better to use directly the epoch and lr arrays and let the user add as much values as we want in it :)